daft/expressions/expressions.py

```diff
@@ -671,6 +676,33 @@ def lengths(self) -> Expression:
         """
         return Expression._from_pyexpr(self._expr.list_lengths())
 
+    def get(self, idx: int | Expression, default: object = None) -> Expression:
+        """Gets the element at an index in each list
+
+        Args:
+            idx: index or indices to retrieve from each list
+            default: the default value if the specified index is out of bounds
+
+        Returns:
+            Expression: an expression with the type of the list values
+        """
+        idx_expr = Expression._to_expression(idx)
+        default_expr = lit(default)
+        return Expression._from_pyexpr(self._expr.list_get(idx_expr._expr, default_expr._expr))
+
+
+class ExpressionStructNamespace(ExpressionNamespace):
+    def field(self, name: str) -> Expression:
```

Comment:
Do you think maybe `.get()` might be more pythonic? (Since dicts use `.get` syntax)

Reply:
I think that makes sense. I'll change it to get.